twilio/rest/accounts/v1/credential/aws.py

```diff
@@ -272,7 +254,7 @@ def delete(self):
         :returns: True if delete succeeds, False otherwise
         :rtype: bool
         """
-        return self._version.delete('delete', self._uri)
+        return self._version.delete(method='DELETE', uri=self._uri, )
```

Comment:
Seeing a lot of changes not (directly) related to header support. Can these be split out?

Reply:
They _can_ but I'm not they _should_. Lots of the changes come from refactoring lots of duplicates and very near duplicates in yoyodyne to simplify the process of generating action functions. If I were the split out the changes, I likely wouldn't pass the SonarQube check on the yoyo side of things.

Reply:
What about just excluding the `header_support` flag?

Reply:
Oh, you mean push the refactor first _then_ the header support. Yeah I can totally do that.

Reply:
https://github.com/twilio/twilio-python/pull/502